webapp/src/components/ActivityPage/Transaction/TransactionDetail/TransactionDetail.tsx

```diff
@@ -29,7 +29,18 @@ const getHref = (tx: Transaction) => {
 }
 
 const TransactionDetail = (props: Props) => {
-  const { asset, text, tx } = props
+  const { asset, text, tx, isCrossChain = false } = props
+  const [txLink, setTxLink] = useState(getHref(tx))
+  useEffect(() => {
+    ;(async () => {
+      if (isCrossChain) {
+        const { AxelarProvider } = await import(
+          'decentraland-transactions/crossChain'
+        )
+        setTxLink(await AxelarProvider.getTxLink(tx.hash))
```

Comment:
The await is not required, right?
```suggestion
        setTxLink(AxelarProvider.getTxLink(tx.hash))
```

Reply:
good call, not needed!